README.md

```diff
@@ -62,10 +62,10 @@ Check out [plugins documentation](./docs/plugins.md).
 First, clone the repo via git:
 
 ```bash
-git clone https://github.com/KELiON/cerebro.git cerebro
+$ git clone https://github.com/KELiON/cerebro.git cerebro
```

Comment:
@lex111 agreed, you make a fair point, but in this case, we expect the fellow developers to already know that `git clone` is a command meant to be run in a terminal, and we choose the easier option for them.

Moreover, the dollar sign is only used in some Linux distros. Others may use something like `>` or the lambda sign or something. Or just a plain space after the directory path. You never know. So it's simpler to take the least common denominator and provide just what is needed: the command.